shared/shot.js

```diff
@@ -181,16 +178,15 @@ function deepEqual(a, b) {
   return true;
 }
 
-function makeUuid() {
-  // FIXME: not a proper uuid
+function makeRandomId() {
+  // Note: this isn't for secure contexts, only for non-conflicting IDs
   let id = "";
   while (id.length < 12) {
-    // 46656 == Math.pow(36, 3)
     let num;
     if (! id) {
-      num = Date.now() % 46656;
+      num = Date.now() % Math.pow(36, 3);
     } else {
-      num = Math.floor(Math.random() * 46656);
+      num = Math.floor(Math.random() * Math.pow(36, 3));
```

Comment:
We only want the ids for clips generated within a single shot to not conflict.  This isn't used for any security-conscious situation.  Things like clip image URLs are generated with UUIDs on the server.